app/views/admin/claims/_answer_section.html.erb

```diff
@@ -0,0 +1,19 @@
+<% unless heading.nil? %>
+  <h2 class="govuk-heading-m">
+    <%= heading %>
+  </h2>
+<% end %>
+
+<dl class="govuk-summary-list govuk-!-margin-bottom-9">
+  <%- answers.each do |(label, answer, slug)| %>
+    <div class="govuk-summary-list__row">
+      <dt class="govuk-summary-list__key">
+        <%= label %>
+      </dt>
+
+      <dd class="govuk-summary-list__value">
+        <%= answer %>
+      </dd>
+    </div>
+  <% end %>
+</dl>
```

Comment:
Is this basically the same as the teacher view? Can we share it?

Reply:
I'd rather not, becuase the teacher view has slightly different markup (two `<dd>` elements instead of one), and they both have different audiences, so best to keep them apart I reckon